### ISSUE TITLE
label style is different in training and test data

### ISSUE THREAD

**Issue by neptune4year** [Yuliang-Liu/Curve-Text-Detector#48]
I find that the label in the training set is usually one polygon for one word, but that in the test set is one polygon for one whole line.
Is that true? Do you have correct labels?
Please refer to image 0468 in training set and image 1233 in test set.

**Comment by zhangshuaitao** [comment]
@neptune4year, all annotations are based on text lines.The question you said should be a special case. We will try to ensure that all annotations are based on text lines. Thanks for your attention.
